### ISSUE TITLE
[MU4 Issue] MSCZ files listed as "MuseScore compressed backup score" in Linux file managers

### ISSUE THREAD

**Issue by MarcSabatella** [musescore/MuseScore#15922]
**Describe the bug**
On Linux Debian at least, MuseScore files get listed as "MuseScore compressed backup score" in file managers, including the file dialogs within MuseScore, due to a wrong MIME type setting

**To Reproduce**
1. In MuseScore, File / Open
2. Navigate if necessary to a folder containing MSCZ files
3. See error: all MSCZ files listed as "MuseScore compressed backup score"

**Expected behavior**
File type should say "MuseScore score"

**Screenshots**
![image](https://user-images.githubusercontent.com/1799936/211895739-f48e8c36-0a01-4da7-8bd1-7c6d6b2b8dd2.png)

**Platform information**
 - OS: Linux

**Additional context**
The problem appears to be incorrect info in the MIME type definition:

https://github.com/musescore/MuseScore/blob/master/build/Linux%2BBSD/musescore.xml.in

Both "*.mscz" and "*.mscz," (with comma) are givne the same MIME type: "x-musescore".  Similarly for ".mscx" and ".mscx," which both get "x-musescore+xml".  Apaprently, at least some implementations of MIME don't allow multiple entries within the same file to have the same MIME type, and the last one wins.

I think the solution is probably to change the MIME type of the backups to be "x-musescore-backup" and "x-musescore-backup+xml" or something like that.  Or maybe someone more expert in MIME can comment or suggest a better solution.  But I can confirm that making this change locally on my system fixes it for me.

**Comment by shoogle** [comment]
Another option would be to remove these registrations because MuseScore is not actually able to open these files until they have been renamed to remove the comma from the extension.
<img width="300px" src="https://user-images.githubusercontent.com/11011881/212788262-6f7e3121-6500-4cb0-903a-72dd64de593c.png"/>

**Comment by MarcSabatella** [comment]
True, although *is* nice seeing the backup files listed with descriptions in the file manager.  Not sure it’s possible to do that without also enabling one to open with MuseScore though.

**Comment by shoogle** [comment]
I think it is possible on Linux if you give them a different MIME type to the proper ones (e.g. `x-musescore-backup` like you suggested earlier), and then you __don't__ specify the new MIME type in MuseScore's `.desktop` file.

I don't think it is possible on Windows or macOS, but I could be wrong.

**Comment by Lazerbeak12345** [comment]
Related to #19718

**Comment by Lazerbeak12345** [comment]
I also noticed that the backup files in muscore 4 are postpended by a `~` instead of a `,` on linux

**Comment by creativecoder** [comment]
I ran into this issue yesterday and opened https://github.com/musescore/MuseScore/pull/34428 to address it.
